src/core/mac/sub_mac.hpp

```diff
@@ -540,11 +540,16 @@ class SubMac : public InstanceLocator, private NonCopyable
 
 #if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
     /**
-     * The SSED sample window in units of 10 symbols.
+     *
      *
      */
```

Comment:
Remove this comment block? Or some comments about this enum block is about CSL parameters?